packages/invoices/src/OutboundShipment/DetailView/reducer.ts

```diff
@@ -191,14 +189,30 @@ export const reducer = (
             lines[existingLineIdx] = {
               ...lines[existingLineIdx],
               ...line,
+              id: lines[existingLineIdx]?.id || line.id,
```

Comment:
In the `UpsertLine` action, can I suggest this:
```
          const existingLineIdx = lines.findIndex(
            ({ id }) => !!id && id === line.id
          );
```

currently, all invoice lines have a blank id because they are new lines - which means that when adding more than one, only the one gets added. any subsequent lines are considered changes to the first one